src/nanotron/data/nanoset.py

```diff
@@ -48,6 +50,9 @@ def __init__(
         # Init
         self.dataset_folders = dataset_folders
         self.sequence_length = sequence_length
+        self.eos_token_id = eos_token_id
+        self.return_positions = return_positions
```

Comment:
we can add a small comment here before merging explaining that return_positions here means we get position_ids from nanoset/datatrove